### PR TITLE
Emandate doc correction

### PR DESCRIPTION
**Pull request by ankitdas13**
Doc reference : https://razorpay.com/docs/api/payments/recurring-payments/emandate

### CODE REVIEW

File: documents/emandate.md [hunk 1]
```diff
@@ -63,20 +72,33 @@ Please refer this [doc](https://razorpay.com/docs/api/recurring-payments/emandat
...
+| customer["name"]*          | string | Customer's name.           |
+| customer["email"]*          | string | Customer's email address.           |
+| customer["contact"]*          | string | Customer's phone number.          |
 | type*        | string  | In this case, the value is `link`.                      |
 | currency*        | string  | The 3-letter ISO currency code for the payment. Currently, only `INR` is supported. |
 | amount*         | integer  | The payment amount in the smallest currency sub-unit.                 |
 | description*    | string  | A description that appears on the hosted page. For example, `12:30 p.m. Thali meals (Gaurav Kumar`).                                                             |
-| subscription_registration           | array  | Details of the authorization payment.                      |
-| notes           | array  | A key-value pair                                                             |
+| subscription_registration["method"]           | string  | The authorization method. In this case, it will be `emandate`.|
```
Comment (sofiya02) [inline documents/emandate.md:89]: All parameters listed [here] are supported

File: documents/emandate.md [hunk 2]
```diff
@@ -36,19 +36,28 @@ $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@raz
...
-| receipt         | string  | Your system order reference id.                                              |
-| notes           | array  | A key-value pair                                                             |
-| token           | array  | A key-value pair                                                             |
+| amount*   | integer      | The amount to be captured (should be equal to the authorized amount, in paise) |
+| currency*   | string  | The currency of the payment (defaults to INR)  |
+| customerId*   | string      | The id of the customer to be fetched |
+| method*      | string  | Payment method used to make the registration transaction. Possible value is `emandate`.  |
+| receipt      | string  | Your system order reference id.  |
+| payment_capture*  | boolean  | Indicates whether payment status should be changed to captured automatically or not. Possible values: true - Payments are captured automatically. false - Payments are not captured automatically. |
+| token["auth_type"]  | string  | Possible value is `netbanking`, `debitcard` or `aadhaar`|
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/emandate.md]: reverted back and added link there

File: documents/emandate.md [hunk 3]
```diff
@@ -36,19 +36,28 @@ $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@raz
...
+| token["max_amount"]  | integer  | Use to set the maximum amount per debit request. The value can range from `500` - `1000000000` (1cr, default value)  |
+| token["expire_at"]  | integer | The timestamp, in Unix format, till when the  registration link should expire |
+| token["notes"]  | object  | A key-value pair  |
+| bank["account_number"]  | string  | Customer's bank account number.  |
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/emandate.md]: bank parameter inside token ,  added link for bank param

File: documents/emandate.md (hunk 1 above)
Comment (ankitdas13) [reply documents/emandate.md]: added param link for subscription_registration

**Review by sofiya02 — APPROVED** [review]
LGTM

**Pull request merged**